src/main/java/uk/gov/justice/hmpps/prison/api/model/questionnaire/CreateIncidentTypeConfigurationRequest.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.hmpps.prison.api.model.questionnaire
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.hmpps.prison.repository.jpa.model.Questionnaire
+
+data class CreateIncidentTypeConfigurationRequest(
```

Comment:
I've made the request json payload the same as the DPS version is stored. e.g. https://github.com/ministryofjustice/hmpps-incident-reporting/blob/main/server/reportConfiguration/types/SELF_HARM_1.ts